modules/build.gradle

```diff
@@ -31,7 +31,7 @@ ext {
 		"com.liferay.gradle.plugins.css.builder.version": "5.0.9",
 		"com.liferay.gradle.plugins.theme.builder.version": "2.0.17",
 		"com.liferay.portal.tools.bundle.support.version": "3.7.4",
-		"com.liferay.portal.tools.service.builder.version": "1.0.460",
+		"com.liferay.portal.tools.service.builder.version": "1.0.462",
```

Comment:
this change to fix the proejct tempate ci test error. Because the gradle workspace plugin has been used 462 verssion. If we don't chagne, it will cause maven service-buider version is not same as gradle

Reply:
we have a test case that compare the service-buider verison between maven and gradle in CI server for project template test.